ethstore/src/json/crypto.rs

```diff
@@ -81,7 +82,7 @@ impl<'a> Visitor<'a> for CryptoFieldVisitor {
 			"kdf" => Ok(CryptoField::Kdf),
 			"kdfparams" => Ok(CryptoField::KdfParams),
 			"mac" => Ok(CryptoField::Mac),
-			_ => Err(Error::custom(format!("Unknown field: '{}'", value))),
+			_ => Ok(CryptoField::Ignored),
```

Comment:
I don't agree with this change. There is definitely a finite number of special cases. Please handle all of them and describe why they exist

Reply:
:+1:

Reply:
done